### PR TITLE
[es] New Page ``` MDN/Writing_guidelines/Page_structures/Page_types/HTTP_header_page_template ```

### PR DESCRIPTION
**Pull request by torresgol10**
Description
New page in Spanish

Motivation
Contribute to open source and Spanish speech

Related to https://github.com/mdn/translated-content/issues/14993

### CODE REVIEW

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: Plantilla de página de encabezado HTTP
+slug: MDN/Writing_guidelines/Page_structures/Page_types/HTTP_header_page_template
+---
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:4]:
```suggestion
slug: MDN/Writing_guidelines/Page_structures/Page_types/HTTP_header_page_template
l10n:
  sourceCommit: cb1c745168764c4646631e7c4289319d782cc83b
---
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+> ```md
+> ---
+> title: <NombreDelElemento>
+> slug: Web/SVG/Element/NombreDelElemento
+> page-type: svg-element
+> status:
+>   - experimental
+>   - deprecated
+>   - non-standard
+> browser-compat: svg.elements.NombreDelElemento
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:26]: Maybe this is from the last template translation

```suggestion
> title: NombreDeLaCabecera
> slug: Web/HTTP/Headers/NameOfTheHeader
> page-type: http-header
> status:
>   - experimental
>   - deprecated
>   - non-standard
> browser-compat: path.to.feature.NameOfTheHeader
```

**Review by Graywolf9 — CHANGES_REQUESTED** [review]
Hi! Thank you for your time and help ! 

I'm looking that some texts are from other template instead of header template, also tell you that we use "cabecera" for this kind of properties and "encabezado" for text content and it seems that you use an assisted translation that change some macros

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md (hunk 2 above)
Comment (torresgol10) [reply files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:26]: Fue un error al copiarlo de otra traducción

**Comment by torresgol10** [comment]
Buenas @Graywolf9 ,

¿Tengo que hacer algún cambio más?

Saludos.

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+>   - non-standard
+> browser-compat: path.to.feature.NameOfTheHeader
+> ---
+> ```
+>
+> - **title**
+>   - : El título que se muestra en la parte superior de la página. Debe tener el formato _NombreDeLaCabecera_. Por ejemplo, la cabecera [Cache-Control](/es/docs/Web/HTTP/Headers/Cache-Control) tiene un _título_ de `Cache-Control`.
+> - **slug**
+>   - : El final de la ruta de URL después de `https://developer.mozilla.org/es/docs/`.
+>     Esto se formateará como `Web/SVG/Element/NombreDelElemento`.
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:36]:
```suggestion
>     Esto se formateará como `Web/HTTP/Headers/NameOfTheHeader`.
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+Si el encabezado tiene muchas directivas disponibles, siéntete libre de incluir múltiples listas de definiciones, subsecciones y explicaciones según corresponda.
+
+## Ejemplos
+
+Nota que usamos el plural "Ejemplos" incluso si la página solo contiene un ejemplo.
+
+### Un título descriptivo
+
+Cada ejemplo debe tener un título H3 (###) que nombre el ejemplo. El título debe ser descriptivo de lo que está haciendo el ejemplo. Por ejemplo, "Un ejemplo simple" no dice nada sobre el ejemplo y por lo tanto, no es un buen título. El título debe ser conciso. Para una descripción más larga, usa el párrafo después del título.
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:130]:
```suggestion
### Un encabezado descriptivo

Cada ejemplo debe tener un encabezado H3 (###) que nombre el ejemplo. El encabezado debe ser descriptivo de lo que está haciendo el ejemplo. Por ejemplo, "Un ejemplo simple" no dice nada sobre el ejemplo y por lo tanto, no es un buen encabezado. El encabezado debe ser conciso. Para una descripción más larga, usa el párrafo después del encabezado.
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+Consulta nuestra guía sobre cómo agregar [ejemplos de código](/es/docs/MDN/Writing_guidelines/Page_structures/Code_examples) para obtener más información.
+
+> **Nota:** A veces querrás enlazar a ejemplos dados en otra página.
+>
+> **Escenario 1:** Si tienes algunos ejemplos en esta página y algunos más en otra página:
+>
+> Incluye un título H3 (###) para cada ejemplo en esta página y luego un título H3 (###) final con el texto "Más ejemplos", debajo del cual puedes enlazar a los ejemplos en otras páginas. Por ejemplo:
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:138]:
```suggestion
> Incluye un encabezado H3 (###) para cada ejemplo en esta página y luego un encabezado H3 (###) final con el texto "Más ejemplos", debajo del cual puedes enlazar a los ejemplos en otras páginas. Por ejemplo:
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+> Ejemplo de Fetch
+>
+> ### Más ejemplos
+>
+> Enlaces a más ejemplos en otras páginas
+> ```
+>
+> **Escenario 2:** Si solo tienes ejemplos en otra página y ninguno en esta página:
+>
+> No agregues ningún título H3; simplemente agrega los enlaces directamente debajo del título H2 "Ejemplos". Por ejemplo:
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:154]:
```suggestion
> No agregues ningún encabezado H3; simplemente agrega los enlaces directamente debajo del encabezado H2 "Ejemplos". Por ejemplo:
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+
+_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._
+
+## Compatibilidad con el navegador
+
+`\{{Compat}}`
+
+_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._
+
+## Ver también
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:174]:
```suggestion
## Véase también
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+> Para ejemplos de esta API, consulta [la página sobre fetch()](https://example.org).
+> ```
+
+## Especificaciones
+
+`\{{Specifications}}`
+
+_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._
+
+## Compatibilidad con el navegador
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:168]:
```suggestion
## Compatibilidad con navegadores
```

**Review by Graywolf9 — COMMENTED** [review]
Hola! Practicamente son los mismos comentarios mencionados anteriormente, los dejo comentados los aplico y hago el _merge_

Muchas gracias por toda tu ayuda, apoyo y paciencia

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+Idealmente, esto debería ser una o dos oraciones cortas.
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">Tipo de cebera</th>
+      <td>
+        Incluye la categoría de la cabecera (o categorías), por ejemplo,
+        {{Glossary("Request header", "Cabecera de solicitud")}},
+        {{Glossary("Response header", Cabecera de respuesta)}},
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:82]:
```suggestion
        {{Glossary("Response header", "Cabecera de respuesta")}},
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+> - `\{{Deprecated_Header}}` — esto genera un banner de **Obsoleto** que indica que el uso de la tecnología está [desaconsejado](/es/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#deprecated).
+>   Si no lo está, puedes eliminar la llamada a la macro.
+> - `\{{Non-standard_Header}}` — esto genera un banner de no estándar que indica que la función no forma parte de ninguna especificación.
+>   Debe actualizar o eliminar las siguientes macros de acuerdo con los siguientes consejos:
+> - `\{{httpsidebar}}` — esto genera el menú lateral de HTTP que debe aparecer en cada página de referencia HTTP. Recuerde eliminar la macro {{MDNSidebar}} cuando copie esta página.
+>   No proporcione macros de encabezado de estado manualmente. Consulte ["Cómo agregar o actualizar estados de funciones"](/es/docs/MDN/Writing_guidelines/Page_structures/Feature_status#how_to_add_or_update_feature_statuses). para agregar estos estados a la página.
+>
+> Se muestran muestras de los banners **Experimental** y **Obsoleto** justo después de este bloque de nota.
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:66]:
```suggestion
> - `\{{Non-standard_Header}}` — esto genera un banner de no estándar que indica que la función no forma parte de ninguna especificación.
>
>   Debe actualizar o eliminar las siguientes macros de acuerdo con los siguientes consejos:
>
> - `\{{httpsidebar}}` — esto genera el menú lateral de HTTP que debe aparecer en cada página de referencia HTTP. Recuerde eliminar la macro {{MDNSidebar}} cuando copie esta página.
>
>   No proporcione macros de encabezado de estado manualmente. Consulte ["Cómo agregar o actualizar estados de funciones"](/es/docs/MDN/Writing_guidelines/Page_structures/Feature_status#how_to_add_or_update_feature_statuses). para agregar estos estados a la página.
>
> Se muestran muestras de los banners **Experimental** y **Obsoleto** justo después de este bloque de nota.
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+        <a href="/es/docs/Web/HTTP/Headers">Cabecera de pista del cliente</a>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">{{Glossary("Forbidden header name", "Nombre de cabecera prohibido")}}</th>
+      <td>sí o no</td>
+    </tr>
+    <tr>
+      <th scope="row">
+        {{Glossary("Cabecera de respuesta permitida por CORS")}}
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:92]:
```suggestion
        {{Glossary("CORS-safelisted response header", "Cabecera de respuesta permitida por CORS")}}
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+Cada ejemplo debe tener un encabezado H3 (###) que nombre el ejemplo. El encabezado debe ser descriptivo de lo que está haciendo el ejemplo. Por ejemplo, "Un ejemplo simple" no dice nada sobre el ejemplo y por lo tanto, no es un buen encabezado. El encabezado debe ser conciso. Para una descripción más larga, usa el párrafo después del encabezado.
+
+Consulta nuestra guía sobre cómo agregar [ejemplos de código](/es/docs/MDN/Writing_guidelines/Page_structures/Code_examples) para obtener más información.
+
+> **Nota:** A veces querrás enlazar a ejemplos dados en otra página.
+>
+> **Escenario 1:** Si tienes algunos ejemplos en esta página y algunos más en otra página:
+>
+> Incluye un encabezado H3 (###) para cada ejemplo en esta página y luego un encabezado H3 (###) final con el texto "Más ejemplos", debajo del cual puedes enlazar a los ejemplos en otras páginas. Por ejemplo:
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:136]:
```suggestion
> Incluye un encabezado H3 (`###`) para cada ejemplo en esta página y luego un encabezado H3 (`###`) final con el texto "Más ejemplos", debajo del cual puedes enlazar a los ejemplos en otras páginas. Por ejemplo:
```

File: files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+
+_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._
+
+## Véase también
+
+Incluye enlaces a páginas de referencia y guías relacionadas con el encabezado HTTP actual. Para obtener más pautas, consulta la [sección Ver también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.
+
+- enlace1
+- enlace2
+- external_link (año)
```
Comment (Graywolf9) [inline files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md:178]:
```suggestion
- enlace_externo (año)
```

**Review by Graywolf9 — APPROVED** [review]

**Pull request merged**